.code-workspace

```diff
@@ -0,0 +1,12 @@
+{
+	"settings": {
+		"ansible.lightspeed.suggestions.enabled": true,
+		"ansible.lightspeed.enabled": true
+	},
+	"folders": [
```

Comment:
What if we remove folders at all?
Just not to hardcode `/projects` here.

Reply:
`folders` block will be added automatically when workspace is created and we'll have unstashed changes in the workspace